src/displayBackend/wayland/Seat.cpp

```diff
@@ -127,8 +127,7 @@ void Seat::readName(const std::string& name)
 
 void Seat::init()
 {
-	mWlSeat = static_cast<wl_seat*>(wl_registry_bind(getRegistry(), getId(),
-									&wl_seat_interface, cVersion));
+	mWlSeat = static_cast<wl_seat*>(bind(&wl_seat_interface));
```

Comment:
how do you use `*? " *" or "* " or "*"?`

Reply:
"* "